tox.ini

```diff
@@ -14,7 +14,7 @@ description =
   Run the tests under {basepython} and
   devel: ansible devel branch
   ansible29: ansible 2.9
-  core: ansible-base 2.10
+  core: ansible-core 2.11+
```

Comment:
looks like `py` already incorporates this.

Reply:
In fact `py` was a mistake and I removed it.